### PR TITLE
Update example on docs/howto/connection/index.rst

### PR DESCRIPTION
**Pull request by mik-laj**
Commented inline.

CC: @rootcss 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: docs/howto/connection/index.rst [hunk 1]
```diff
@@ -118,62 +118,53 @@ The JSON format contains an object where the key contains the connection ID and
...
-      port:
+      port: null
       schema: airflow
-
-    CONN_B:
+    druid_broker_default:
       conn_type: druid
       extra: '{"endpoint": "druid/v2/sql"}'
       host: druid-broker
-      is_encrypted: false
```
Comment (mik-laj) [inline docs/howto/connection/index.rst:164]: These fields do not exist in the output file.

Comment (rootcss) [reply docs/howto/connection/index.rst:164]: Ah! missed that. I had removed both of those fields in later version. Thanks 👍

**Review by rootcss — APPROVED** [review]
Looks good.

**Comment by rootcss** [comment]
@mik-laj tests have failed due to missing indent in tests' connections dict. https://github.com/apache/airflow/pull/10236/checks?check_run_id=960763585#step:6:1028

**Comment by mik-laj** [comment]
@rootcss Thanks you. I updated this PR now.

**Review by rootcss — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**